cf-check/repair.c

```diff
@@ -127,11 +128,24 @@ int repair_main(int argc, const char *const *const argv)
 
 int repair_default()
 {
+    // This function is used by cf-execd and cf-agent, not cf-execd
```

Comment:
I think this line needs a consistency check.

Reply:
Haha, good catch. 

> Compilers don't read comments. Programmers don't read comments.